src/Universum/String/Conversion.hs

```diff
@@ -293,6 +295,15 @@ be safe to collapse.
 readEither :: (ToString a, Read b) => a -> Either Text b
 readEither = first toText . Text.Read.readEither . toString
 
+-- | Polymorhpic version of 'Text.Read.readMaybe'.
+--
+-- >>> readMaybe @Text @Int "123"
+-- Just 123
+-- >>> readMaybe @Text @Int "aa"
+-- Nothing
+readMaybe :: (ToString a, Read b) => a -> Maybe b
```

Comment:
Very good to see this change, thanks :+1: 

Maybe it would be worth going with another order of type arguments?:

```suggestion
readMaybe :: forall b a. (ToString a, Read b) => a -> Maybe b
```

I think specifying the resulting type is more often necessary, and this order of arguments will cause fewer breaking changes.

But this is only my opinion.